########################################
# Audio frame in frequency domain (with real and imaginary parts)
########################################

Header header

uint32 frameLength
uint32 nChannels
uint32 fs
float32[] data